### PR TITLE
Updated schemas and began inserting data into pipeline

### PR DESCRIPTION
**Pull request by najjajm**

### CODE REVIEW

File: churchland_pipeline/acquisition.py [hunk 1]
```diff
@@ -13,20 +13,23 @@ class Session(dj.Manual):
...
     """
 
     class User(dj.Part):
         definition = """
         -> master
         -> lab.User
+        ---
         """
 
     class Notes(dj.Part):
```
Comment (shenshan) [inline churchland_pipeline/acquisition.py:27]: We could potentially put this into the master table if there is only one note for each entry.

File: churchland_pipeline/acquisition.py [hunk 2]
```diff
@@ -47,46 +58,35 @@ class EphysRecording(dj.Imported):
     definition = """
     -> Session
     ---
-    ephys_file_path: varchar(512) # file path
-    ephys_file_name: varchar(256) # file name
+    ephys_file_path: varchar(1012) # file path (temporary until issues with filepath attribute are resolved)
     """
 
-    class Meta(dj.Part):
+    class AcquisitionParams(dj.Part):
```
Comment (shenshan) [inline churchland_pipeline/acquisition.py:64]: Consider putting into the master table.

File: churchland_pipeline/acquisition.py [hunk 3]
```diff
@@ -47,46 +58,35 @@ class EphysRecording(dj.Imported):
...
     definition = """
     -> Session
     ---
-    speedgoat_file_path: varchar(512) # file path
-    speedgoat_file_name: varchar(256) # file name
+    speedgoat_file_path: varchar(1012) # file path (temporary until issues with filepath attribute are resolved)
     """
 
-    class Meta(dj.Part):
+    class AcquisitionParams(dj.Part):
```
Comment (shenshan) [inline churchland_pipeline/acquisition.py:84]: same

File: churchland_pipeline/acquisition.py [hunk 4]
```diff
@@ -101,16 +101,16 @@ class EmgChannelGroup(dj.Imported):
     ---
     -> ephys.EmgElectrode
     emg_channel_group : blob # array of channel numbers corresponding to EMG data
-    emg_channel_notes : varchar(1024) # notes for the channel set
+    emg_channel_notes : varchar(4095) # notes for the channel set
     """
 
-    class Corrupted(dj.Part):
+    class Sortable(dj.Part):
```
Comment (shenshan) [inline churchland_pipeline/acquisition.py:107]: master table?

File: churchland_pipeline/processing.py [hunk 5]
```diff
@@ -65,7 +109,7 @@ class Template(dj.Part):
 class Neuron(dj.Imported):
     definition = """
     # Sorted neuron
-    -> acquisition.NeuralChannels
+    -> acquisition.NeuralChannelGroup
     neuron_id : smallint unsigned # unique unit ID
     ---
     -> NeuralSpikeSorter
```
Comment (shenshan) [inline churchland_pipeline/processing.py:115]: Only one version of clustering result?

File: churchland_pipeline/processing.py [hunk 6]
```diff
@@ -38,7 +82,7 @@ class NeuralSpikeSorter(dj.Lookup):
 class MotorUnit(dj.Imported):
     definition = """
     # Sorted motor unit
-    -> acquisition.EmgChannels
+    -> acquisition.EmgChannelGroup
     motor_unit_id : smallint unsigned # unique unit ID
     ---
     -> EmgSpikeSorter
```
Comment (shenshan) [inline churchland_pipeline/processing.py:88]: one version of clustering result?

File: churchland_pipeline/processing.py [hunk 7]
```diff
@@ -77,7 +121,7 @@ class SessionSpikes(dj.Part):
         # Full session spike indices
         -> master
         ---
-        neuron_spike_indices : longblob # array of spike indices
+        neuron_session_spikes : longblob # array of spike indices
```
Comment (shenshan) [inline churchland_pipeline/processing.py:124]: put into master table?

File: churchland_pipeline/tasks/pacman.py [hunk 8]
```diff
@@ -81,28 +102,55 @@ class Trial(dj.Part):
...
+# -------------------------------------------------------------------------------------------------------------------------------    
+@schema
+class Emg(dj.Imported):
+    definition = """
+    # raw, trialized, and aligned EMG data
+    -> acquisition.EmgChannelGroup
+    -> AlignmentIndices
+    emg_channel : tinyint unsigned # channel number (indexed relative to EMG channel group)
+    ---
+    emg_channel_data : longblob # channel data
```
Comment (dimitri-yatsenko) [inline churchland_pipeline/tasks/pacman.py:145]: `_data` is rather vague.   What exactly is being stored? E.g. `emg_voltage_signal: longblob # (mV)`

File: howtos.py [hunk 9]
```diff
@@ -0,0 +1,6 @@
+# Dropping a schema
```
Comment (dimitri-yatsenko) [inline howtos.py:1]: You might like to add
```python
dj.list_schemas()   #list all schema available to me
```

**Review by dimitri-yatsenko — APPROVED** [review]

File: churchland_pipeline/processing.py [hunk 10]
```diff
@@ -7,27 +7,71 @@
 # LEVEL 0
 # -------------------------------------------------------------------------------------------------------------------------------
 
+@schema
```
Comment (shenshan) [inline churchland_pipeline/processing.py:10]: I'm thinking about putting tables in this module into ephys.

File: churchland_pipeline/tasks/pacman.py [hunk 11]
```diff
@@ -17,15 +17,36 @@ class ArmPosture(dj.Lookup):
...
+        [6,   'InSecondPad'],
+        [7,   'PastLastDot'],
+        [100, 'Success'],
+        [251, 'Glitch'],
+        [252, 'Abort'],
+        [253, 'Failure'] 
+    ]
     
 @schema
 class Task(dj.Imported):
```
Comment (shenshan) [inline churchland_pipeline/tasks/pacman.py:52]: change as we discussed yesterday.

**Review by shenshan — APPROVED** [review]
Hi Najja, impressive work! Left some comments for your reference. I'll need to understand better how the alignment (sync) is done.

**Pull request merged**